CONTRIBUTING.md

```diff
@@ -8,6 +8,14 @@ In order to accept your pull request, we need you to [submit a CLA](https://gith
 
 By contributing to Trino, you agree that your contributions will be licensed under the [Apache License Version 2.0 (APLv2)](LICENSE).
 
+# Go Test
```

Comment:
This is not related to other changes in the PR. Can you squash all commits but keep this in a separate one, to make reviewing easier?